packages/vue-components/src/Tab.vue

```diff
@@ -13,81 +16,82 @@
 </template>
 
 <script>
-import {toBoolean} from './utils/utils.js'
+import { toBoolean } from './utils/utils';
 
 export default {
   props: {
     header: {
-      type: String
+      type: String,
+      default: '',
     },
     disabled: {
       type: Boolean,
-      default: false
-    }
+      default: false,
+    },
   },
   computed: {
-    headerRendered () {
-      return this.$refs.header.innerHTML
+    headerRendered() {
+      return this.$refs.header.innerHTML;
     },
-    active () {
-      return this._tabset.show === this
+    active() {
+      return this._tabset.show === this;
     },
-    index () {
-      return this._tabset.tabs.indexOf(this)
+    index() {
+      return this._tabset.tabs.indexOf(this);
     },
-    show () {
-      return this._tabset && this._tabset.show === this
+    show() {
+      return this._tabset && this._tabset.show === this;
     },
-    transition () {
-      return this._tabset ? this._tabset.effect : null
+    transition() {
+      return this._tabset ? this._tabset.effect : null;
+    },
+    disabledBool() {
+      return toBoolean(this.disabled);
     },
-    disabledBool () {
-      return toBoolean(this.disabled)
-    }
   },
-  created () {
-    this._ingroup = this.$parent && this.$parent._tabgroup
-    let tabset = this
-    while (tabset && tabset._tabset!==true && tabset.$parent) {
-      tabset = tabset.$parent
+  created() {
+    this._ingroup = this.$parent && this.$parent._tabgroup;
+    let tabset = this;
+    while (tabset && tabset._tabset !== true && tabset.$parent) {
+      tabset = tabset.$parent;
     }
     if (!tabset._tabset) {
-      this._tabset = {}
-      console.warn('Warning: "tab" depend on "tabset" to work properly.')
+      this._tabset = {};
+      console.warn('Warning: "tab" depend on "tabset" to work properly.');
     } else {
-      tabset.tabs.push(this)
+      tabset.tabs.push(this);
       if (!this._ingroup) {
-        tabset.headers.push(this)
-      } else {
-        if (!~tabset.headers.indexOf(this.$parent)) {
-          tabset.headers.push(this.$parent)
-        }
+        tabset.headers.push(this);
+      // eslint-disable-next-line no-bitwise
+      } else if (!~tabset.headers.indexOf(this.$parent)) {
+        tabset.headers.push(this.$parent);
       }
-      this._tabset = tabset
+      this._tabset = tabset;
     }
     if (this._ingroup) {
-      this.$parent.tabs.push(this)
+      this.$parent.tabs.push(this);
     }
   },
-  beforeDestroy () {
-    if (this._tabset.active === this.index) { this._tabset.active = 0 }
+  beforeDestroy() {
+    if (this._tabset.active === this.index) { this._tabset.active = 0; }
     if (this._ingroup) {
-      var index = parent.tabs.indexOf(this);
```

Comment:
👀  should this have been referencing `$parent`?

Reply:
Yup seems like it should have been `$parent` as the function `parent.tabs.indexOf` does not exist.